build_docs.py

```diff
@@ -425,7 +350,7 @@ def translation_branch(locale_repo, locale_clone_dir, needed_version: str):
     It could be enhanced to return tags, if needed, just return the
     tag as a string (without the `origin/` branch prefix).
     """
-    git_clone(locale_repo, locale_clone_dir)
+    Git(locale_repo, locale_clone_dir).clone()
     remote_branches = run(["git", "-C", locale_clone_dir, "branch", "-r"]).stdout
```

Comment:
Should we run this through the `Git` abstraction?

Reply:
Done, went a tad further by passing the git abstraction as a parameter of this function.